docs/markdown/ios/toast.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Toast
+description: Toasts are brief and small messages that overlay content, but do not block the user’s flow, as they are out of the way and ephemeral. Toasts do not require user action and primarily acknowledge that a user has performed an action or completed a task.
+fullwidth: true
+---
+
+<ImgContainer src="https://i.pinimg.com/originals/27/32/0d/27320d8b7731c7cea23fbf917e7d2160.jpg" noPadding alt="an example of toast"/>
+
+## Usage guidelines
+
+<TwoCol>
+  <Group>
+    <Do title="When to use" />    
```

Comment:
Double check that these all have periods at the end

Reply:
Done